bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ThingTypeConverter.java

```diff
@@ -95,12 +97,22 @@ protected ThingTypeXmlResult unmarshalType(HierarchicalStreamReader reader, Unma
         ThingTypeXmlResult thingTypeXmlResult = new ThingTypeXmlResult(
                 new ThingTypeUID(super.getUID(attributes, context)), readSupportedBridgeTypeUIDs(nodeIterator, context),
                 super.readLabel(nodeIterator), super.readDescription(nodeIterator), readCategory(nodeIterator),
-                getListed(attributes), getChannelTypeReferenceObjects(nodeIterator), getProperties(nodeIterator),
+                getListed(attributes), getExtensibleChannelTypeIds(attributes),
+                getChannelTypeReferenceObjects(nodeIterator), getProperties(nodeIterator),
                 getRepresentationProperty(nodeIterator), super.getConfigDescriptionObjects(nodeIterator));
 
         return thingTypeXmlResult;
     }
 
+    protected List<String> getExtensibleChannelTypeIds(Map<String, String> attributes) {
+        String extensible = attributes.get("extensible");
+        if (extensible == null) {
+            return Collections.emptyList();
+        }
+
+        return Arrays.asList(extensible.split(","));
```

Comment:
Should we maybe also trim the strings?
Keep in mind us poor human beings who were ~~forced~~ taught in school to waste paper with spaces like this `extensible="alarm, brightness, foo, bar"`

Reply:
I did actually thought about that. In the XSD I did specify a pattern which does not allow spaces. In case the XML is validated against the XSD this will fail early. But we may also add trimming, better save then sorry :-)